x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/index.js

```diff
@@ -214,7 +215,7 @@ InnerCustomPlot.propTypes = {
 
 InnerCustomPlot.defaultProps = {
   formatTooltipValue: p => p.y,
-  tickFormatX: undefined,
+  tickFormatX: x => moment(x).format('HH:mm'),
```

Comment:
Does this affect the "response time" and "requests per minute" charts? I think they just use `xType=time` and expect the default react-vis formatting for x-ticks. Overriding the default `tickFormatX` might affect this.

The react-vis formatting is pretty smart about whether to show years/months/days/minutes/hours/seconds etc depending on the range and such. There might be good reasons for re-implementing this logic, eg. fixing the [timezone issues we discovered recently](https://github.com/elastic/kibana/issues/47959). But this doesn't seem to match what we already have out of the box from react-vis.

Reply:
My bad, this shouldn't have been committed. It's from investigating the timezone issue you described. Will remove.